test/parallel/test-startup-empty-regexp-statics.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 const common = require('../common');
-const assert = require('assert');
-const { spawnSync, spawn } = require('child_process');
+const assert = require('node:assert');
+const { spawnSync, spawn } = require('node:child_process');
```

Comment:
this seems like it's out of scope for the current PR, and should be done everywhere or not at all